its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -442,4 +444,27 @@ void supportMavenEncryption() throws Exception {
     ORCHESTRATOR.executeBuild(build);
   }
 
+  @Test
+  void supportMavenEncryptionWithDefaultSecuritySettings() throws Exception {
+
+    wsClient.settings().set(new SetRequest().setKey("sonar.forceAuthentication").setValue("true"));
+    wsClient.users().create(new CreateRequest().setLogin("julien2").setName("Julien2").setPassword("123abc"));
+
+    MavenBuild build = MavenBuild.create(ItUtils.locateProjectPom("maven/maven-only-test-dir"))
+            .setGoals(cleanSonarGoal());
+
+    File securityXml = new File(this.getClass().getResource("/security-settings.xml").toURI());
+    File settingsXml = new File(this.getClass().getResource("/settings-with-encrypted-sonar-password.xml").toURI());
+
+    String userHomeDir = System.getProperty("user.home");
+    Path defaultPath = Paths.get(userHomeDir, ".m2", "settings-security.xml");
+    Files.copy(securityXml.toPath(), defaultPath);
+
+    build.addArgument("--settings=" + settingsXml.getAbsolutePath());
+
+    build.setProperty("sonar.login", "julien");
+    build.addArgument("-Psonar-password");
+    ORCHESTRATOR.executeBuild(build);
```

Comment:
What is the assertion that we are trying to verify with this test? That the build does not fail? Is it enough for the build not to throw an error to be considered successful?

Reply:
Yes, if password wasn't decrypted correctly, you fail with authorization error. Maybe I can rewrite it to use assertDoesn'tThrow or something like this (also concerns the test above)